msbuild/Makefile

```diff
@@ -373,6 +379,11 @@ all-local:: $(MSBUILD_PRODUCTS) .stamp-test-xml
 
 # make all the target assemblies build when any of the sources have changed
 $(foreach dll,$(MSBUILD_TASK_ASSEMBLIES),$(dll)/bin/$(CONFIG)/$(TARGETFRAMEWORK)/$(dll).dll $(dll)/bin/$(CONFIG)/$(TARGETFRAMEWORK)/$(dll).pdb): .build-stamp
+$(foreach dll,$(TRANSLATED_ASSEMBLIES),Xamarin.Localization.MSBuild/bin/$(CONFIG)/$(TARGETFRAMEWORK)/$(dll).resources.dll): .build-stamp
+
+y: $(foreach dll,$(TRANSLATED_ASSEMBLIES),Xamarin.Localization.MSBuild/bin/$(CONFIG)/$(TARGETFRAMEWORK)/$(dll).resources.dll)
```

Comment:
Debugging spam?

Reply:
maybe @rolfbjarne can look at it again, but these changes are in the makefile changes he sent me. https://github.com/rolfbjarne/xamarin-macios/commit/eb167b344647f60c21ada79120a8cc701473850a#diff-14d3973c482862a4cc5f2663d281bbb6R382